.github/workflows/stale-pr.yaml

```diff
@@ -3,8 +3,14 @@ on:
   schedule:
     - cron: "12 3 * * *" # arbitrary time not to DDOS GitHub
 
+permissions:
+  contents: read
+
 jobs:
   stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+      pull-requests: write  # for actions/stale to close stale PRs
```

Comment:
This looks sensible but I think we don't have a good way to test it, so I am afraid this PR could break this and we wouldn't notice in a long time